src/style/cell-line.module.css

```diff
@@ -1,11 +1,37 @@
 .container {
-    border: 1px black solid;
+    box-sizing: border-box;
+    display: grid;
+    grid-template-columns: auto 1fr;
+    width: 100%;
+    margin: 0 auto;
+    gap: 40px;
+    margin-bottom: 40px;
+}
+
+.left-card {
+    width: 460px;
```

Comment:
are these really necessary? can't we reuse the style sheet from the disease subpage?